Assets/AppCenter/Plugins/iOS/Core/Utility/NSStringDictionaryHelper.mm

```diff
@@ -27,7 +27,7 @@
   NSMutableDictionary<NSString *, NSString *> *nsdictionary = [[NSMutableDictionary alloc] initWithCapacity:count];
   for (int i = 0; i < count; ++i) {
     NSString *key = [NSString stringWithUTF8String:keys[i]];
```

Comment:
Let's use our convert method for the keys too

Reply:
Fixed